lms/resources/_js_config/__init__.py

```diff
@@ -118,14 +118,29 @@ def enable_oauth2_redirect_error_mode(
 
         :param error_details: Technical details of the error
         :type error_details: str
-        :param auth_url: URL for the "Try again" button in the dialog
-        :type auth_url: str
+        :param auth_route: route for the "Try again" button in the dialog
+        :type auth_route: str
```

Comment:
```suggestion
        :param auth_route: route for the "Try again" button in the dialog
```

Are we not doing these?

Reply:
They were just pre-existing but in this case I think they are actually kind of useful because the method accepts two strings, one bool and one list of strings